src/main/java/dev/jbang/cli/Info.java

```diff
@@ -304,7 +343,73 @@ class Jar extends BaseInfoCommand {
 	@Override
 	public Integer doCall() throws IOException {
 		ScriptInfo info = getInfo(false);
-		System.out.println(info.applicationJar);
+		out.println(info.applicationJar);
+		return EXIT_OK;
+	}
+}
+
+@CommandLine.Command(name = "docs", description = "Open the documentation file in the default browser.")
+class Docs extends BaseInfoCommand {
+
+	@Option(names = {
+			"--open" }, negatable = true, defaultValue = "false", description = "Open the (first) documentation file/link in the default browser")
+	public boolean open;
+
+	@Override
+	public Integer doCall() throws IOException {
+
+		ScriptInfo info = getInfo(false);
+
+		ProjectFile[] toOpen = new ProjectFile[1];
+
+		if (info.description != null) {
+			out.println(info.description);
+		}
+
+		info.docs.forEach((String id, List<ProjectFile> docs) -> {
+			out.println(ConsoleOutput.yellow(id + ":"));
+			docs.forEach(doc -> {
+
+				String uripart = doc.backingResource == null || Util.isURL(doc.originalResource) ? doc.originalResource
+						: Paths.get(doc.backingResource).toUri().toString();
+				String suffix = doc.backingResource != null ? "" : " (not found)";
+
+				if (toOpen[0] == null && "main".equals(id) && doc.backingResource != null) {
+					toOpen[0] = doc;
+				}
+
+				out.printf("  %s%s%n", uripart, suffix);
+
+			});
+		});
+
+		if (open) {
+			if (GraphicsEnvironment.isHeadless()) {
+				Util.infoMsg("Cannot open documentation file in browser in headless mode");
+			} else {
+
+				if (toOpen[0] != null) {
+					try {
+						Desktop.getDesktop().browse(getDocsUri(toOpen[0]));
+					} catch (IOException e) {
+						Util.infoMsg("Documentation file to open not found: " + toOpen[0]);
+					}
+				} else {
+					Util.infoMsg("No documentation file to open found");
+				}
+			}
+		} else {
+			Util.infoMsg("Use --open to open the documentation file in the default browser.");
+		}
```

Comment:
```suggestionif (!open) {
            Util.infoMsg("Use --open to open the documentation file in the default browser.");
			return EXIT_OK;
        }
        if (GraphicsEnvironment.isHeadless()) {
            Util.infoMsg("Cannot open documentation file in browser in headless mode");
			return EXIT_OK;
        }

        if (toOpen[0] == null) {
            Util.infoMsg("No documentation file to open found");
			return EXIT_OK;
        }
        try {
            Desktop.getDesktop().browse(getDocsUri(toOpen[0]));
        } catch (IOException e) {
            Util.infoMsg("Documentation file to open not found: " + toOpen[0]);
        }
```

@quintesse do you think a more assertive style in the ifs would help readability?

Reply:
Sure, it avoids excessive nesting in a place where nesting doesn't really do anything useful.

Reply:
Fixed by 9ee8eb06490d398f8272136c583a96044830f70a